test/math.js

```diff
@@ -439,6 +439,14 @@ describe('Math', function () {
       expect(Object.is(-0, Math.round(-0.5 + Number.EPSILON / 3.99))).to.equal(true);
       expect(Object.is(-0, Math.round(0 - Number.EPSILON / 3.99))).to.equal(true);
     });
+
+    it('returns 1 / Number.EPSILON + 1 for 1 / Number.EPSILON + 1', function () {
+      expect(Math.round(1 / Number.EPSILON + 1)).to.equal(1 / Number.EPSILON + 1);
```

Comment:
This all is true for `double-precision 64-bit binary format IEEE 754` with `round to nearest, ties to even` mode, which are used by ECMAScript (see https://people.mozilla.org/~jorendorff/es6-draft.html#sec-ecmascript-language-types-number-type)